modules/ROOT/pages/download-and-install-service-mesh.adoc

```diff
@@ -17,6 +17,21 @@ To download the Anypoint Service Mesh client, type the following command at the
 
 The Anypoint Service Mesh client is now downloaded on your file system.
 
+== Configure Your Environment Variable
+
+Before you begin installing Anypoint Service Mesh, update your `PATH` environment variable to store system-wide values, such as the directories to search for the executable programs.
+
+To configure your `PATH` variable on Linux: 
+
+. Open the `.bashrc` file in your home directory.
++
+For example, type `cd /home/<your-user-name>/.bashrc`) in a text editor.
```

Comment:
.bashrc`**)** > the parenthesis is intended to be there? If so, it's missing the opening sign: **(**

Reply:
Good catch - thank you!